Source/JavaScriptCore/tools/JSDollarVM.cpp

```diff
@@ -63,6 +63,7 @@
 #include "VMTrapsInlines.h"
 #include "WasmCapabilities.h"
 #include <unicode/uversion.h>
+#include <wtf/MemoryPressureHandler.h>
```

Comment:
Please alphabetize this.

Reply:
It is alphabetical already. The "w" from "wtf" comes after "u" from "unicode".